Sources/SwiftProtobufPluginLibrary/Descriptor.swift

```diff
@@ -755,12 +746,12 @@ public final class FieldDescriptor {
   /// Extensions can be declared within the scope of another message. If this
   /// is an extension field, then this will be the scope it was declared in
   /// nil if was declared at a global scope.
-  public private(set) weak var extensionScope: Descriptor?
+  public private(set) unowned var extensionScope: Descriptor?
 
   /// When this is a message field, the message's `Desciptor`.
-  public private(set) weak var messageType: Descriptor!
+  public private(set) unowned var messageType: Descriptor!
   /// When this is a enum field, the enum's `EnumDesciptor`.
-  public private(set) weak var enumType: EnumDescriptor!
+  public private(set) unowned var enumType: EnumDescriptor!
```

Comment:
I wonder if these should be regular optional instead of IUOs?

Reply:
yup, on my list of cleanups (regrets) to do as part of 2.0.